code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -1,5 +1,6 @@
 /obj/machinery/portable_atmospherics/scrubber
 	name = "portable air scrubber"
+	desc = "It's a small portable scrubber, capable of sucking glasses from its surroundings. It has an internal tank, and a slot for inserting an external tank. It can be wrenched to connection ports to pump and withdraw gasses from the internal tank. "
```

Comment:
`glasses` again. Change `sucking`. Remove the extra space at the end of the string.